src/ua.c

```diff
@@ -1413,9 +1343,7 @@ int ua_options_send(struct ua *ua, const char *uri,
 	if (!dialbuf)
 		return ENOMEM;
 
-	err = ua_uri_complete(ua, dialbuf, uri);
-	if (err)
-		goto out;
+	mbuf_write_str(dialbuf, uri);
```

Comment:
perhaps `uri` can be used directly here ?

Reply:
Yes, will use.  Looks much cleaner then.